examples/graphene_apps/cppopenvino/README.md

```diff
@@ -73,6 +73,72 @@ https://creativecommons.org/licenses/by/4.0/
 
   `docker-compose -f docker-compose.yaml -f docker-compose-sgx.yaml up`
 
+## Testing OpenVino Object detection with Python worker
+
+### Testing OpenVino (without Avalon)
+
+- For non SGX mode, execute below steps from [cppopenvino](http://github.com/hyperledger/avalon/tree/master/examples/graphene_apps/cppopenvino) directory
+
+  - Build Openvino python workload image
+    `docker-compose -f docker-compose-pw.yaml build`
+  - Run Openvino inference workloads in python worker
+    `docker-compose -f docker-compose.yaml -f docker-compose-pw.yaml -f compose/ov-subnet.yaml up`
+
+- For SGX mode, execute below steps
+
+  - Build Openvino python workload image
+    `docker-compose -f docker-compose-pw.yaml build`
+  - Run Openvino inference workloads in graphene python worker
+    `docker-compose -f docker-compose.yaml -f docker-compose-sgx.yaml -f docker-compose-pw.yaml -f test/python_worker/compose/graphene-sgx.yaml -f test/python_worker/compose/ov-subnet.yaml up`
```

Comment:
why are we keeping `graphene-sgx.yaml` in `test/python_worker/compose/graphene-sgx.yaml` instead of   `compose/graphene-sgx.yaml`

Reply:
It was kept inside python_worker/compose to indicate, this compose file has to be used while testing openvino with python_worker

Reply:
moved graphene-sgx.yaml from cppopenvino/python_worker/compose to cppopenvino/compose and renamed to python-worker-gsgx.yaml